scripts/plugins/config/parser.py

```diff
@@ -17,9 +17,15 @@ def get_config_list(config_file, schema_file):
     \n\t PLUGIN SCHEMA FILE PATH:           [{schema_file}]    \
     \n\n"
     )
-
-    schema_yaml = load_yaml(schema_file)
-    config_yaml = load_yaml(config_file)
+    try:
+        schema_yaml = load_yaml(schema_file)
+        config_yaml = load_yaml(config_file)
+    except FileNotFoundError as e:
+        logger.error(
+            f"Required config file was not found. \
+                To fix this please add the following file: [{e.filename}]"
+        )
+        sys.exit(2)
```

Comment:
run_cmd try catch blocks exit with 1xx, should we consider making this a 1xx exit as well? What denotes a 2 vs 101